R/themes.R

```diff
@@ -93,16 +88,19 @@ bs_theme_set <- function(theme) {
   invisible(old_theme[["bootstraplib_theme"]])
 }
 
-
-bs_theme_create <- function(version = version_default()) {
+#' @rdname theming
+#' @export
+bs_theme_create <- function(version = version_default(), bootswatch = NULL) {
```

Comment:
Now that I'm working on a new (functional) interface, I think I'm going to change this to `bs_theme()` (and add more arguments), so I'd prefer that we don't export until that happens